src/content-handlers/iiif/modules/uv-pdfcenterpanel-module/PDFCenterPanel.ts

```diff
@@ -197,11 +195,9 @@ export class PDFCenterPanel extends CenterPanel<
       }
 
       this._render(this._pageIndex);
-    });
-
-    this._$zoomOutButton.onPressed((e: any) => {
-      e.preventDefault();
+    }, true, true);
```

Comment:
Maybe I'm missing something, but why do you need the anchor as button setting in two places in this file, when these are buttons now and were divs in the past?

Reply:
This should be resolved now